test/system_test.py

```diff
@@ -141,13 +141,11 @@ def test_install_py_typing_extensions():
 
 
 def test_install_pytorch_fortran():
-    spack_install(
```

Comment:
Please revert the changes for this package.
The dev of this packages recommends to build the c-part with gcc, but the fortran-interface needs to be with nvhpc for icon compatability.